### ISSUE TITLE
[Bug] User Input does not get processed

### ISSUE THREAD

**Issue by mellobacon** [mellobacon/Termello#20]
**Describe the bug**
Running programs that require input does not work

**To Reproduce**
1. Run Termello.
2. Try to run a console app that requires input. (Example is python or node)
3. Either the program that is run will throw an error or it will just not input anything and the prompt will show. Most likely the error is not in your program but in Termello. A good way to check is to run your program in another terminal. If it works, its Termello.

**Expected behavior**
Processing user input needs to work

**Desktop (please complete the following information):**
 - Windows, both versions

**Additional context**
The problem shows in this function https://github.com/mellobacon/Termello/blob/056f53702c59eea100826da090eef6e144c12122/src/components/terminal.js#L50
You will see the way I have processing input is a bit hacky. I need a better way of doing this so that when you execute something that requires user input it reacts accordingly.

I am aware of this issue, but because of the lack of knowledge I have not been able to fix the problem. If you came across this, feel free to make a PR if you think you have a fix :)

**Comment by anf** [comment]
I have put a bit of research into this, seems like you only need to redirect stdin, not sure how though.

**Comment by mellobacon** [comment]
Yep thats the same issue I had. Most likely with the way processing inputs are implemented it might not be possible without a rewrite. Thanks for researching though.

**Comment by anf** [comment]
@mellobacon, I highly doubt that it's not possible without a rewrite. I had a discussion with a colleague and he said that `stdin.write` should do it, I think it'd be wise to give it a bit of time.
